services/scan/src/precinct_scanner_app.test.ts

```diff
@@ -120,6 +119,14 @@ function postTemplate(
     .expect(200, { status: 'ok' });
 }
 
+async function setAppPrecinct(app: Application, precinctId?: PrecinctId) {
+  await put(app, '/precinct-scanner/config/precinct', {
+    precinctSelection: precinctId
+      ? singlePrecinctSelectionFor(precinctId)
+      : ALL_PRECINCTS_SELECTION,
+  });
+}
```

Comment:
I realized we don't have a test that confirms that unconfiguring the scanner using the `DELETE /precinct-scanner/config/election` endpoint also clears the precinct config. Reading the code, I _think_ it works, but might be worth checking/testing if you haven't already, since the frontend doesn't explicitly clear the precinct config on unconfigure.

Reply:
I don't think we have a test that covers the` DELETE /precinct-scanner/config/election` endpoint at all. Will do some manual testing.

Reply:
We should also of course write a test, but I need to close this out and need to do manual testing anyway.